.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Bug report
+about: Report a bug to fix vapoursynth.
+title: "...!"
+labels: bug
+assignees: myrsloik
+
+---
+
+**Provide enough details:**
+* What's the *problem*?
+* Your *environment*?
```

Comment:
This issue states a few things that are of interest (especially points 4 and 5):
https://github.com/vapoursynth/vapoursynth/issues/455#issuecomment-507051404

1. Which OS
2. Which Version of VapourSynth are you using?
3. How did you install VapourSynth? Which package-manager, etc... Is this the latest version provided by the vendor, etc...

Reply:
Updated with more details in newest commit.